language/evm/stdlib/sources/Evm.move

```diff
@@ -1,5 +1,5 @@
 /// Module which provides access to EVM functionality, including information about the executing transaction.
-////
+///
```

Comment:
I don't know why but removing this could help this failing test case pass https://github.com/move-language/move/runs/8219288895?check_suite_focus=true . I don't understand how comments affect the results. It would be great someone could give me some lights.